### PR TITLE
[[ Bug 16845 ]] Reinstate missing paint tools

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 1]
```diff
@@ -1817,6 +1817,40 @@ function revIDEGraphicTools
    return tReturnData
 end revIDEGraphicTools
 
+function revIDEGraphicToolControllers
+   local tTooltip, tName, tCount
+   add 1 to tCount
+   put "com.livecode.tool.FillColor" into tName
+   put "Graphic Fill Color" into tTooltip
+   updateOptionArray tName, "graphic fill color tool", tTooltip, tCount, tReturnData
```
Comment (peter-b) [inline Toolset/libraries/revidelibrary.8.livecodescript:1825]: Should there be a `local tReturnData` declaration?

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 2]
```diff
@@ -8694,12 +8754,27 @@ on revIDECreateTransientTextField pRect, pCallback, pCallbackParams, pCurrentTex
...
    set the text of tLongID to pCurrentText
+   
+   -- Ensure the text fits
+   if the formattedWidth of tLongID > the width of tLongID then
+      set the width of tLongID to the formattedWidth of tLongID
+   end if
+   set the height of tLongID to the formattedHeight of tLongID
+   set the loc of tLongID to tLoc
+   set the left of tLongID to tLeft
+
```
Comment (peter-b) [inline Toolset/libraries/revidelibrary.8.livecodescript]: Are these changes to `revIDECreateTransientTextField` an intentional part of this PR?  I can't seem to spot anything in here which uses transient fields...

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 3]
```diff
@@ -1826,57 +1860,83 @@ those paint tools.
...
-   put 2 into tReturnData["com.livecode.tool.Brush"]["order"]
-
-   return tReturnData
+   local tDataA
+   
+   local tTooltip, tName, tCount
+   
+   add 1 to tCount
+   put "com.livecode.tool.Pencil" into tName
+   put "Pencil Tool - Draws a freehand line on an image" into tTooltip
```
Comment (peter-b) [inline Toolset/libraries/revidelibrary.8.livecodescript]: @livecodeali What do you think about standardising on imperative voice for these tooltips? E.g. "Draw a freehand line on an image", "Fill an outline in an image", "Select a rectangle area of an image," and so on?

File: Toolset/libraries/revidelibrary.8.livecodescript (hunk 2 above)
Comment (livecodeali) [inline Toolset/libraries/revidelibrary.8.livecodescript]: Crept in from here https://github.com/livecode/livecode-ide/pull/1016/files I think

**Comment by peter-b** [comment]
@livecode-vulcan review ok 9276242

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 92762426ee43b52edad3cb11a0f944807772ad70

**Pull request merged**